docs/DeveloperGuide.md

```diff
@@ -929,18 +929,31 @@ Therefore, we plan to improve the UI by **adding a horizontal scroll bar** so th
+    * In the future version, we plan to integrate the 2 commands into 1 command: `pay hr/HOURS_PAID | amount/AMOUNT`.
+        * To be specific, the new `pay` command accepts either `hr/HOURS_PAID` or `amount/AMOUNT` but not both and there must be exactly one argument given.
+        * `hr/HOURS_PAID` specifies the amount the student **pays**, and the amount of `HOURS_PAID * RATE` will be added to the paid amount.
+        * `amount/AMOUNT` specifies the amount the student **repays**, which will be subtracted from the owed amount then added to the paid amount.
+    * By integrating the 2 features into 1 command, the user can focus on reading the instructions of 1 command and choosing which option they want instead of trying one of them then finding out that it is not what they want. <br>
+
+1. **Allow negative HOURS_PAID and negative AMOUNT for the new `pay` command `pay hr/HOURS_PAID | amount/AMOUNT`:** Currently, UGTeach only allow **positive multiples of 0.5** for the hours specified in the `pay command`. Hence, if users have used the `pay command` on mistake,
    e.g. keyed in the wrong number of hours, the only way for user to revert back to the previous amount paid by the student is through remembering the previous amount paid, and edit the student's `paid` field using the `edit` command.
    This might be inconvenient for the user, as the user might not remember the previous amount that was paid by the student.
    Therefore, we plan to allow **negative multiples of 0.5** for the hours specified in the `pay` command. If users were to make a mistake in the `pay command`, he can enter the same `pay command`, but with the negative hours specified.
    e.g. User typed `pay 1 hr/2` wrongly, when he wants to increase the paid amount by the student by 3 hours instead.
    He can first type `pay 1 hr/-2` to 'undo' the previous `pay command`, and type `pay 1 hr/3` this time for the correct update.
    Special cases that we handle:
-    * hr/0 will still not be accepted.
+    * `hr/0` will still not be accepted.
     * When the resulting paid amount of the student that user want to update is less than 0, the command will not be executed and an error message will be shown.
-      The main purpose for allowing negative hours for `pay command` is to allow user to 'undo' his mistakes made due to him specifying the wrong number of hours paid by the student.
-      Hence, the resulting paid amount from the execution of the `pay command` should not be negative in any daily use case.<br><br>
+     
+    Similarly, we plan to also allow **negative number with at most 2 decimal places** as a valid AMOUNT when the user need to 'undo' the previous settlement of the owed amount.
```

Comment:
needs to `undo`

Reply:
I deliberately used the quotation mark at first to quote 'undo',
since undo is not an actual command, I think 'undo' would be more logical?

Reply:
haha, i intended to put it in quotation mark

Reply:
let me make it double quoted

Reply:
ohhhhh okay i guess thats a grammar error i missed out